app/serializers/admin_serializers/users_listing_serializer.rb

```diff
@@ -0,0 +1,31 @@
+module AdminSerializers
+  # class AdminIndexSerializer
+  class UsersListingSerializer < ActiveModel::Serializer
```

Comment:
Please add a relevant name as discussed for naming serializers in this project. Module name should be the controller name and serailzer name should be the action name